### PR TITLE
Added  LCA in Binary tree

### PR DESCRIPTION
**Pull request by Omji-Kesharwani**

### CODE REVIEW

**Review by Saloni6111 — COMMENTED** [review]
remove exe file & binary_serch.cpp file its code already present in main

**Comment by Omji-Kesharwani** [comment]
@Saloni6111  Please review it again

**Review by Saloni6111 — COMMENTED** [review]
please remove binary_serch.cpp file its code already present.

**Comment by Saloni6111** [comment]
> please remove binary_serch.cpp file itself not only code

File: C++/binary_search.cpp
Comment (Omji-Kesharwani) [inline C++/binary_search.cpp:1]: I have removed this file

**Comment by Omji-Kesharwani** [comment]
why it has been excluded from hactoberfest

**Pull request merged**